_includes/0.1/left_sidebar.html

```diff
@@ -22,6 +22,10 @@
         <a href="{% link docs/0.1/index.md %}">Introduction</a>
         <a href="{% link docs/0.1/glossary.md %}">Glossary</a>
     </div>
+    <div class="left-sidebar-group">
+        <div class="left-sidebar-header">Grid Features</div>
+        <a href="{% link docs/0.1/grid_product.md %}">Grid Product</a>
```

Comment:
And maybe also remove grid_ and Grid here.

Reply:
I agree about removing "Grid" from the left-nav TOC, so I pushed a fixup commit with this change.

However, the file name should stay as is, per doc best practices (which recommend that the file name matches the topic title, which is "Grid Product").